src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/resource/VideoLinkBookingControllerTest.kt

```diff
@@ -89,12 +90,16 @@ class VideoLinkBookingControllerTest {
     @Bean
     fun locationsInsidePrisonClient() = mock<LocationsInsidePrisonClient>()
 
+    @Bean
+    fun additionalBookingDetailRepository() = mock<AdditionalBookingDetailRepository>()
+
     @Bean
     fun videoLinkBookingsService() = VideoLinkBookingsService(
       videoBookingRepository(),
       referenceCodeRepository(),
       videoAppointmentRepository(),
       locationsInsidePrisonClient(),
+      additionalBookingDetailRepository(),
     )
   }
```

Comment:
What's the reason for these being @Bean declarations? In tests usually enough to declare repos as mocks and construct the service in the test class? Just interested if there's any additional benefit in Beans.

Reply:
See here - https://github.com/ministryofjustice/hmpps-book-a-video-link-api/pull/358

When I made the original change to get rid of the deprecation warnings there was a knock on effect due to a known bug with the replacement Mockito annotations i.e. this is a workaround.